src/main/flush.ts

```diff
@@ -1,6 +1,11 @@
 import fs from 'fs';
 import path from 'path';
-import { getDefaultState, IState } from '../common/constants';
+import {
+  getDefaultState,
+  IPreset,
+  IState,
+  PRESETS_DIR,
+} from '../common/constants';
 
 export const stateToString = (state: IState) => {
```

Comment:
maybe we should call this serializeState?

Reply:
sure, we can but think we should do another PR for that.